src/constructs/cloudwatch/ec2-alarms.test.ts

```diff
@@ -0,0 +1,73 @@
+import "@aws-cdk/assert/jest";
+import { SynthUtils } from "@aws-cdk/assert";
+import { Vpc } from "@aws-cdk/aws-ec2";
+import { Stack } from "@aws-cdk/core";
+import { simpleGuStackForTesting } from "../../utils/test";
+import type { AppIdentity } from "../core/identity";
+import { GuApplicationLoadBalancer } from "../loadbalancing";
+import { Gu5xxPercentageAlarm } from "./ec2-alarms";
+
+const vpc = Vpc.fromVpcAttributes(new Stack(), "VPC", {
+  vpcId: "test",
+  availabilityZones: [""],
+  publicSubnetIds: [""],
+});
+
+const app: AppIdentity = {
+  app: "testing",
+};
+
+describe("The Gu5xxPercentageAlarm construct", () => {
+  it("should create the correct alarm resource with minimal config", () => {
+    const stack = simpleGuStackForTesting();
+    const alb = new GuApplicationLoadBalancer(stack, "ApplicationLoadBalancer", { ...app, vpc });
```

Comment:
tangent: I'm refactoring `GuApplicationLoadBalancer` to update it's logicalId override implementation and have been thinking the `id` can be defaulted and removed from the constructor as it's always `ApplicationLoadBalancer`.